notebooks/Field Apps Deployment Using Python.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>What do you think of adding a section at the bottom to generate a url scheme for Field Maps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>